lyonjs/shortvid.io
# feat(sidebar): add responsive

## 🤔 Why do you want to make those changes?

To allow a better experience on mobile devices, some responsive is required

## 🧑‍🔬 How did you make them?

I created a mobile version of the sidebar with a burger menu and handled the style with media queries to work on mobile.

## 🧪 How to check them?

- check the preview
- check the code
